### ISSUE TITLE
kubegres / postgresql monitoring and alerting

### ISSUE THREAD

**Issue by michbsd** [reactive-tech/kubegres#117]
Hi, 

Any recommendations for monitoring/alerting on kubegres/postgresql health? 

We have been using pg-x in the past, but looking for something better.

Any recommendations and/or experiences would be much appreciated. 

Many thanks,

**Comment by hxtk** [comment]
I'm in the process of setting up kubegres and this is something I'm going to have to deal with in the future, so I'm wondering if you've found kubegres-specific issues with this that prevent tools like the Postgres Prometheus Exporter [1] from being used?

https://github.com/prometheus-community/postgres_exporter

**Comment by michbsd** [comment]
Nope.. not yet.. still using pg-x - even though we are having many resource issues with it

**Comment by michbsd** [comment]
hi @alex-arica 
Can you recommend something? What are you using for monitoring the state of your kubegres deployment? 

Many thanks,
Michael
